doc/user/content/overview/open-cdc.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Materialize Open CDC"
+description: "The Open CDC format allows Materialize to detect when records are duplicated or out of order."
+menu:
+  main:
+    parent: "overview"
+    weight: 4
+---
+
+Change data capture (CDC) tools provide feeds that record any changes to a database. Typically, the feeds are then saved to another platform, like Kafka, for storage or processing.
+
+However, sometimes the stream can have missing or duplicate records, or send records out of order. For example, if a CDC tool crashes while writing a record, it may retry and write the record again, resulting in a duplicate entry.
+
+To prevent this kind of error, Materialize has developed the **Open CDC format**, a format for the updates that CDC tools propagate to other systems.  The Open CDC format is designed especially to prevent record duplication or missequencing by including enough information for a downstream data consumer (like Materialize) to recognize when records are missing, duplicated, or out of order. Each record contains:
+
+- the changes in a record (the updated fields or the deletion or addition of a new record)
```

Comment:
I think this is probably "the record itself". At least, in the thing we've implemented that is what it is. So if we are changing the 7th field in a record, we present the whole record (twice actually, once +1 and once -1). There is a good reason for this, even though it feels like maybe we should only present the changed bits.